pkg/clients/digitalocean.go

```diff
@@ -165,3 +168,25 @@ func IgnoreNotFound(err error, response *godo.Response) error {
 	}
 	return err
 }
+
+// GetResourceID gets the resource ID in int form for a given string.
+func GetResourceID(id string) int {
+	result, err := strconv.Atoi(id)
+
+	if err != nil {
+		return -1
+	}
+
+	return result
+}
+
+// GetName Used the get the name of the resource and an ID for it if one exists.
+// If no ID is present it will be -1
+func GetName(o metav1.Object, oName string) string {
```

Comment:
It doesn't appear this function is actually used.

Reply:
Removed!